### DISCUSSION TITLE
Does it possible to send an auto response mail?

### DISCUSSION THREAD

**Discussion by litonmazumder** [PHPMailer/PHPMailer#2566] [Q&A]
Hello!

I would like to send an auto-response mail to my client which I will get from ($_POST['email']) and also I get the client information from the input field in my email. So does it possible to send in both ways Using Phpmailer. Thank You.

**Comment by Synchro** [comment]
It sounds like you're describing [a contact form](https://github.com/PHPMailer/PHPMailer/blob/master/examples/contactform.phps). That example sends an email to you, but you can do something very similar to send a "thank you" or whatever to the submitter as well. That said, you need to be extremely careful with how you do that as it's very easy to accidentally make a spam gateway that will be exploited.

The important thing if you do do that is to make sure that you *only* either:

* Send a message containing user-supplied content to a fixed address.
* Send a message containing fixed content to a user-supplied address.

The other two possible combinations are either useless or dangerous!

**Reply by litonmazumder** [reply]
Thanks for your reply. Actually, I would like to send an email to my client when he/she submits a form. I am not just giving a thank you message when the form is submitted. I would like to receive an email with input details and also send my client to a mail with details what I will provide or whatever it is. It would be vise versa. Hope you understand.

**Reply by Synchro** [reply]
Yes, you can send messages to whoever you like, containing whatever you like, but if you make a form that allows submitting both the email address a message is sent to, and even the tiniest bit of content in subject or message body, it's a spam gateway and will be ruthlessly exploited, so I advise you not to do that.
